lib/ay_tests/image_builder.rb

```diff
@@ -210,5 +212,22 @@ def change_boot_order
     def backup_image
       system "sudo virt-clone -o #{IMAGE_NAME} -n #{IMAGE_NAME}_sav --file /var/lib/libvirt/images/#{IMAGE_NAME}_sav.qcow2"
     end
+
+    # Determine the host IP
+    #
+    # @return [String] Host IP address.
+    #
+    # Taken from Veewee to make sure that the IP matches.
+    def local_ip
+      # turn off reverse DNS resolution temporarily
+      orig, Socket.do_not_reverse_lookup = Socket.do_not_reverse_lookup, true
+
+      UDPSocket.open do |s|
+        s.connect '64.233.187.99', 1 # google
```

Comment:
I am not sure, if we have internet available at all the time. Could be a problem, but lets see. Problems will be solved if they are happen and not before :-)